scripts/db-clone/clone-prod-to-staging.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# This script orchestrates the cloning of Keycloak and Loculus databases from production to staging
+# Keycloak is dumped second and loaded first to prevent potential race conditions
+
+# Set variables
+CHILD_SCRIPT="./clone.sh"
+PROD_KC_DUMP="production_keycloak_dump.sql"
+PROD_LOC_DUMP="production_loculus_dump.sql"
+PROD_KC_DB="pathoplexus_prod_keycloak"
+PROD_LOC_DB="pathoplexus_prod_loculus"
+STAGING_KC_DB="pathoplexus_staging_keycloak"
+STAGING_LOC_DB="pathoplexus_staging_loculus"
+STAGING_KC_USER="staging_keycloak_user"
+STAGING_LOC_USER="staging_loculus_user"
```

Comment:
It doesn't require an username and password for the production database? What are the assumptions / required environment?